test/detection/test_line_counter.py

```diff
@@ -70,3 +73,320 @@ def test_calculate_region_of_interest_limits(
     with exception:
         result = LineZone.calculate_region_of_interest_limits(vector=vector)
         assert result == expected_result
+
+
+@pytest.mark.parametrize(
+    "vector, bbox_sequence, expected_count_in, expected_count_out",
```

Comment:
Please rename the parameters `expected_count_in` and `expected_count_out` to `expected_crossed_in` and `expected_crossed_out` respectively. `count_in` and `count_out` have a well defined meaning in context of `LineZone`.

Reply:
Please do the same for all occurrences.